package/autoyast2.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           autoyast2
-Version:        3.1.151.3
+Version:        3.1.151.3.1
```

Comment:
IMHO we should use:

* 3.2.X for `master`.
* 3.1.X for `SLE-12-SP2`.
* 3.1.X.Y for `SLE-12-SP2-CASP`.

Current version the `SLE-12-SP2` branch is `3.1.156`, so you should use `3.1.151.4` here.